projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -101,6 +101,8 @@ function () {
 		add_action( 'wp_head', array( new Automattic\Jetpack\Social\Meta_Tags(), 'render_tags' ) );
 
 		add_filter( 'jetpack_get_available_standalone_modules', array( $this, 'social_filter_available_modules' ), 10, 1 );
+
+		add_action( 'admin_init', array( $this, 'set_up_sharing_limits' ) );
```

Comment:
I wonder if we could hook this into specific screens rather than every admin call. It probably doesn't matter on the average site, but could cause a problem on large multisites like WPCOM

Reply:
Good idea, I'll look into this.